server/controllers/article_controller.js

```diff
@@ -83,9 +83,9 @@ export const getArticleAnnotations = (user, uri, topLevelOnly) => {
       return article.annotations;
     });
   } else {
-    const deepPath = 'annotations.childAnnotations.childAnnotations.childAnnotations.childAnnotations.childAnnotations.childAnnotations';
+    const deepPath = 'annotations'.concat('.childAnnotations'.repeat(50));
     return getArticle(uri)
-    .deepPopulate(deepPath, { populate: { annotations: { match: query } } })
+    .deepPopulate(deepPath, { match: query })
```

Comment:
Could Peter's problem be coming from this change? I think the syntax that was there was what was specified in the deepPopulate documentation.